packages/components/tsconfig.json

```diff
@@ -13,9 +13,17 @@
         "noEmit": true,
         "declaration": true,
         "outDir": "dist",
+        "rootDir": "./src",
         "paths": {
             "mattermost-redux/*": ["./node_modules/mattermost-redux/src/*"],
-            "@mattermost/types/*": ["./node_modules/@mattermost/types/src/*"]
-        }
+            "@mattermost/types/*": ["./node_modules/@mattermost/types/src/*"],
```

Comment:
I think these paths entries might be able to be removed now since I added the parallel builders